skia/RNLocalizeModule.cpp

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (C) 1994-2022 OpenTV, Inc. and Nagravision S.A.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+#include <cxxreact/JsArgumentHelpers.h>
+
+#include "RNLocalizeModule.h"
+
+using namespace folly;
+namespace facebook {
+namespace xplat {
+
+RNLocalizeModule::RNLocalizeModule() { }
+
+std::map<std::string, folly::dynamic> RNLocalizeModule::getConstants() {
```

Comment:
Is it possible to drop the "using namespace folly;" declaration or can avoid referencing Folly on usage?

Reply:
updated and working without folly namespace.